javascript/photopea.js

```diff
@@ -375,32 +375,19 @@
     }
   }
 
+  let photopeaPopupPopupShown = false;
```

Comment:
Can we rename these variables?
- `photopeaPopupPopupShown` => `photopeaWarningShown`
- `controlnet_disable_photopea_popup` => `controlnet_photopea_warning`

Reply:
sure, I didn't put too much thought in the naming of those variables
I did say feel free to make modifications, 
in case you're not aware you are actually able to push to my Fork because `Allow edits and access to secrets by maintainers` es checked
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/40751091/5097743e-c6bf-416b-8af7-055a94b52747)

also maybe some of my wording of the messages are not as good
if you want to change those go ahead